Software/C/examples/example_3_servo.c

```diff
@@ -1,34 +1,38 @@
 #include "pivotpi.c"
 
+#include <wiringPi.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+
+// gcc example_3_servo.c -lwiringPi -lwiringPiDev -lpthread -lm -lwiringPiPca9685
```

Comment:
Can we document here what this is?  I know this is the command you enter to compile.  But can we make that explicit, and also have a line of instruction on how to run the compiled code?